assets/src/modules/Digitizing.js

```diff
@@ -115,6 +115,8 @@ export class Digitizing {
         // Set draw color to value in local storage if any or default (red)
         this._drawColor = localStorage.getItem(this._repoAndProjectString + '_drawColor') || '#ff0000';
 
+        this._simpleGeometry = false;
```

Comment:
```suggestion
        // Single part or multipart geometry
        this._singlePartGeometry = false;
```

I think I would stick to QGIS terminology "Single parts : Multipart" :  (and other occurences in the code below)

![image](https://github.com/user-attachments/assets/98ae0e84-195c-4fc8-bd62-64c59556ce09)

What do you think ?

Reply:
Done